subprojects/diagnostics/src/main/java/org/gradle/api/reporting/dependencies/HtmlDependencyReportTask.java

```diff
@@ -101,6 +100,7 @@ public DependencyReportContainer reports(Action<? super DependencyReportContaine
      *
      * @since 4.2
      */
+    @Incubating
```

Comment:
I don't think we should mark any `@Inject`ed thing as `@Incubating`. Also, the method should probably be `protected`, but that's a different story.

Reply:
I removed the `@Incubating` and the Javadoc on the method, and made the method protected.